docs/administration-guide/security/user-control.md

```diff
@@ -1,3 +1,49 @@
+# User Authentication
+
 !!! info "Work in Progress"
 
-    This page is under construction.
\ No newline at end of file
+    This page is under construction.
+
+The security layer for Gaffer is currently only enforced by the REST API. We
+recommend restricting users such that they do not have access to the cluster and
+by only allowing users to connect to Gaffer via the REST API.
```

Comment:
Not sure on this sentence phrasing, the bit after and could do with being clearer.